QuizApp/QuizApp/UI/Screens/Leaderboard/LeaderboardList.swift

```diff
@@ -0,0 +1,21 @@
+import SwiftUI
+
+struct LeaderboardList: View {
+
+    let leaderboard: Leaderboard
+
+    var body: some View {
+        ScrollView(.vertical) {
```

Comment:
Vertical is by default, you can remove it https://developer.apple.com/documentation/swiftui/scrollview/init(_:showsindicators:content:)